apps/console/src/app/pages/no-beta-access.tsx

```diff
@@ -0,0 +1,29 @@
+import { useSelector } from 'react-redux'
+import { getUserState } from '@console/domains/user'
+import { RootState } from '@console/store/data'
+import { UserInterface } from '@console/shared/interfaces'
+
+import './no-beta-access.scss'
+import { Link } from '@console/shared/ui'
+
+export function NoBetaAccess() {
+  const user = useSelector<RootState, UserInterface>((state) => getUserState(state))
+  return (
+    <div className="no-beta-access w-full h-screen bg-brand-900 flex items-center justify-center text-element-light-darker-400">
```

Comment:
We didn't use this background color anything, can we add an `bg-brand-500` and on the `no-beta-access__dialog` we can add an `box-shadow: 0px 4px 32px rgba(0, 0, 0, 0.12);` 
What do you think ?